diff --git a/node_modules/pi-wifi/index.js b/node_modules/pi-wifi/index.js
index f1a60c3..6b940ec 100644
--- a/node_modules/pi-wifi/index.js
+++ b/node_modules/pi-wifi/index.js
@@ -17,9 +17,9 @@ const commands = {
   interfaceUp: 'ifup :INTERFACE',
   scan: 'sudo iwlist wlan0 scan | grep ESSID | cut \'"\' -f2',
   startSupplicant: 'sudo wpa_supplicant -Dwext -c :CONFIG_FILE -B -i :INTERFACE  && sudo chattr -i :DNS_FILE',
-  wpaDisconnect: 'wpa_cli disconnect',
+  wpaDisconnect: 'wpa_cli -i wlan0 disconnect',
   wpaInterface: 'wpa_cli interface :INTERFACE',
-  wpaList: 'wpa_cli list_networks'
+  wpaList: 'wpa_cli -i wlan0 list_networks'
 };
 
 
@@ -446,7 +446,7 @@ function listNetworks(callback) {
     if (!err) {
       var networksList = stdout.split('\n');
       networksArray = [];
-      networksList.splice(0, 2); //Remove headers
+      networksList.splice(0, 1); //Remove headers
       networksList.splice(networksList.length - 1, 1); //Remove footer
 
       for (var networkIndex in networksList) {
